assets/component-card.css

```diff
@@ -177,6 +177,14 @@
     transform: scale(1.03);
   }
 
+  .card .card__inner .card__media {
+    z-index: 1;
+  }
+
+  .card .card__inner  {
+    z-index: 2;
```

Comment:
This seems safe for card style cards, but breaks the z order we're relying on for standard card shadows. Shadows need to remain behind adjacent cards.
<img src="https://screenshot.click/10-48-7pill-ezv54.png" width="350" />

Reply:
Good catch Ken! Updated the z-index